perception/autoware_tensorrt_bevdet/README.md

```diff
@@ -0,0 +1,69 @@
+# tensorrt_bevdet <!-- cspell: ignore bevdet -->
+
+## Purpose
+
+tensorrt_bevdet is a dynamic 3D bev object detection package based on 6 surround view cameras.
+
+## Inner-workings / Algorithms
+
+BEVDet is a BEV perception algorithm based on panoramic cameras. It unifies multi-view images into the perspective of BEV for 3D object detection task. In this implementation, BEVDet network to inference with TensorRT.
+
+## Inputs / Outputs
+
+### Inputs
+
+| Name                               | Type                           | Description                         |
+| ---------------------------------- | ------------------------------ | ----------------------------------- |
```

Comment:
The input/output table uses `~/input/` and `~/output/`  paths. While this might be clear within the context of Autoware, it's better to specify the actual topic names without the leading `~/`.  Using absolute topic names (e.g., `/input/topic_img_fl`) makes it more robust and avoids potential ambiguity.